VaticanBAV/et/BAVet176.xml

```diff
@@ -27,12 +30,211 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                         <collection>Aethiopici</collection>
                         <idno>Aeth. 176</idno>
                     </msIdentifier>
+
+                    
+                    <msContents><summary/></msContents>
+                    
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf" n="87">87</measure>
+                                    <dimensions type="outer" unit="mm">
+                                        <height>135</height>
+                                        <width>90</width>
+                                    </dimensions>
+                                </extent>
+                                <collation>
+                                    <note>The catalogue states that there are eleven quires, but only describes the structure of eight.
+                                        The first two folios are protective.
+                                    There are no quire marks.</note>
+                        <!--            <list>
```

Comment:
Why is the quire structure hidden?

Reply:
because I cannot make sense of the catalogue's description (protective quire + eight quires described, but all in all they should be 11 quires). Maybe someone else can interprete it correctly:
Codex duabus partibus constat, quarum prior ff. 3-66 anplectitur, posterior ff. 67-87,
ff. t. 2 pro custodia praemissis. Fasciculi sunt undecim, quorum primus senio, secundus
tertius sextus octavus quiniones, quartus et quintus quaterniones, septimus ternio,
omnes sine notis.

Reply:
I understand. I think you can give the quire structure nevertheless with a note, that the last three quires are not described in the catalogue. As you prefer.